recipe/meta.yaml

```diff
@@ -23,21 +24,21 @@ requirements:
   host:
     - cymem >=2.0.2,<2.1.0
     - cython >=0.25,<3.0
-    - cython-blis >=0.7.8,<0.8.0
+    - cython-blis >=1.0.0,<1.1.0
     - murmurhash >=1.0.2,<1.1.0
-    - numpy {{ numpy }}
+    - numpy >=2.0.0,<2.1.0
     - pip
     - preshed >=3.0.2,<3.1.0
     - python
     - setuptools
     - wheel
   run:
     - python
-    - {{ pin_compatible('numpy') }}
+    - numpy >=2.0.0,<2.1.0
     - murmurhash >=1.0.2,<1.1.0
     - cymem >=2.0.2,<2.1.0
     - preshed >=3.0.2,<3.1.0
-    - cython-blis >=0.7.8,<0.8.0
+    - cython-blis >=1.0.0,<1.1.0
     - wasabi >=0.8.1,<1.2.0
     - srsly >=2.4.0,<3.0.0
     - catalogue >=2.0.4,<2.1.0
```

Comment:
typing_extensions pinning can be removed

Reply:
Yes, thanks